source/plugin/scheduler.rst

```diff
@@ -163,3 +165,139 @@ In addition, there are a few other operations that are safe to do asynchronously
 
 * Independent network requests
 * Filesystem I/O (excluding files used by Sponge)
+
+Compatibility with other libraries
+==================================
+
+.. _ExecutorService: https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html
```

Comment:
From the looks of it, `ExecutorService_` is only ever referenced once. Wouldn't it just be better to do
`ExecutorService <https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ExecutorService.html>`
in the text itself?